microprofile/scheduling/src/test/java/io/helidon/microprofile/scheduling/CronedBean.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c)  2020 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.microprofile.scheduling;
+
+import java.time.LocalTime;
+
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class CronedBean {
```

Comment:
I think you mean `ScheduledBean` or `CronnedBean`, Croned is from word Crone...

Reply:
😀 "a crone is an old woman who may be disagreeable, malicious, or sinister in manner"

Reply:
Exactly ;)

Reply:
fixed